StartingExample/StartingExample.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="GoRogue" Version="2.4.3" />
+    <PackageReference Include="MonoGame.Framework.DesktopGL.Core" Version="3.7.0.7" />
+    <PackageReference Include="SadConsole" Version="8.7.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\src\SadConsole.GoRogue.csproj" />
```

Comment:
If StartingExample is meant to be a "template" of sorts, I would swap this with the NuGet reference. I'll file an issue for that.

Reply:
This can and should definitely be changed once any sort of alpha release has been published to NuGet 😃